### ISSUE TITLE
Tie goes to the sender

### ISSUE THREAD

**Issue by elear** [linuxwolf/ietf-saa#9]
The SAA should exercise caution in their role.  The SAA should avoid engaging in ambiguous circumstances.  Mike St. John's initial response to the call for comments on the SoW was a good example.  As I wrote on list, to me that was a last call comment to a topic that had been hotly debated in plenary, and Mike was inbounds responding to the IETF list.  Without relitigating all of that,  what appears as overly aggressive intervention lands us in meta-discussions about what's appropriate.

**Comment by ekr** [comment]
I think this is an attribution error: the reason that we see meta-discussions about what's appropriate is that people choose to litigate all the edge cases rather than recognizing that the lines are necessarily fuzzy and granting the SAAs a certain amount of discretion to moderate the discussion in the interest of the greater good. Note that I said "a certain amount", not unlimited, and I'm not saying that it's never appropriate to challenge the SAA decision, merely that that, like appeals, should be used sparingly.

**Comment by elear** [comment]
@ekr I disagree that it is an attribution error.  The community needs to be able to govern the use of authority.  But nobody's going to seriously question clear cases of abuse, and those who do will be shot down quickly.  That didn't happen last time because there was a lot of agreement that the SAA crossed the line.  Better that the SAA use their authority sparingly, insofar as we're not completely inundated by abusive people.

**Comment by ekr** [comment]
@elear I don't expect to persuade you here, but I would like to make several points:

1. I don't know about "completely inundated" but I view the ietf@ list as already nearly unreadably toxic. I think that position has  a fair amount of support, as indicated by the discussion at the latest plenary. And fixing that requires giving the moderators a fair amount of discretion.

2.  We do have mechanisms for governing the use of authority: the SAAs are supervised by the IESG, who are subject to nomcom and recall. 

3. I'm not sure that there was a "lot of agreement" that the SAAs had overstepped their authority. Rather there were a relatively small absolute number of people who felt that they had and were very vocal and a lot of people who, for whatever reason, didn't say anything, so we don't know what they think. I can tell you that for myself I felt that they had not overstepped but didn't have the energy to get sucked into the argument (see above about unreadably toxic).

**Comment by elear** [comment]
@ekr It is indeed one of those rare instances where you haven't convinced me (you were right about that part ;-). 

Community governance requires the ability to hold dialog.  When in doubt, the SAA should not stifle that dialog.  That is my point.  "take that to the IESG and the NOMCOM" is simply insufficient, and just a bit insensitive when one considers views of the recall process.

I am not suggesting that we permit a toxic environment.  If the list really is toxic, then the SAA should have a target-rich environment, and not have to enter into questionable cases; or at least shouldn't start with those.  To be fair, we might have a  different understanding of what makes a toxic environment... 

And we should be careful to not avoid a big issue: is the IETF list **the** forum or not?  If it is, then let's decide rules for how people should participate that provide for the sort of expressions we see at the f2fs, but at the same time **do** call for respectful discussion.
